features/steps/fastlane_build_report_steps.rb

```diff
@@ -0,0 +1,77 @@
+Given "fastlane is installed" do
+    $fastlane_installed ||= begin
+      step('I run the script "features/scripts/install-fastlane.sh" synchronously')
+      true
+    end
+  end
+
+  # Default request values. Values not set in scenario remain as nil.
+  @API_KEY = nil
```

Comment:
I think it's more conventional to use lower case for variable names in Ruby - in my relatively limited experience, at least; happy to be shown otherwise.